server/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -356,6 +382,11 @@ public void handleException(TransportException exp) {
             }
         }
 
+        private void handleBlockException(final ClusterBlockException blockException) {
```

Comment:
do we really need this method? can't we just inline it in the two places we call it?